src/FakeItEasy/Configuration/PropertySetterConfiguration.cs

```diff
@@ -101,10 +78,8 @@ public IPropertySetterConfiguration WhenArgumentsMatch(Func<ArgumentCollection,
         }
 
         private static IPropertySetterConfiguration AsPropertySetterConfiguration(
-            IVoidConfiguration voidArgumentValidationConfiguration)
-        {
-            return new ProperySetterAdapter(voidArgumentValidationConfiguration);
-        }
+            IVoidConfiguration voidArgumentValidationConfiguration) =>
+            new ProperySetterAdapter(voidArgumentValidationConfiguration);

```

Comment:
Ditto. If VS doesn't muck it up all the time, I could go for it.

For me (and it may be ReSharper), it keeps getting reformatted to

```c#
private static IPropertySetterConfiguration AsPropertySetterConfiguration(
        IVoidConfiguration voidArgumentValidationConfiguration) =>
    new ProperySetterAdapter(voidArgumentValidationConfiguration);
```

… which I could also live with.